### ISSUE TITLE
How many pieces of data were simulated using ART in the training set

### ISSUE THREAD

**Issue by XLOXL** [MicrobeLab/DeepMicrobes-data#3]
Hi, I was wondering how many pieces of data were simulated using ART in the training set used in Deepmicrobe paper.

**Comment by MicrobeLab** [comment]
Hi, we simulated about 500,000 reads per class as an empirical value, with a lower bound of read number ensuring 2x depth for the largest genome (or a group of genomes within a class). Whether more reads should be simulated and whether the simulated reads should be exhausted depend on specific setting.

**Comment by XLOXL** [comment]
Hi, thank you for your answer! If I simulated about 10,000 reads for each class, training with DeepmMcrobes , do I need to change the parameters of the model?

**Comment by XLOXL** [comment]
Hi, thank you for your answer! If I simulated about 10,000 reads for each class, training with DeepMcrobes , do I need to change the parameters of the model?

**Comment by XLOXL** [comment]
Hi, thank you for your answer! If I simulated about 10,000 reads for each class, training with DeepMicrobes , do I need to change the parameters of the model?

**Comment by MicrobeLab** [comment]
The number of reads needed often depends on the training performance in a specific dataset. It is not required to change hyper-parameters along with the number of reads. But hyper-parameters may need to be tuned given a specific circumstance.

**Comment by XLOXL** [comment]
I understand, thank you very much for your answer!

**Comment by XLOXL** [comment]
I would like to ask you about the 150bp dataset of MAGs data set, the subsequence with the same label in the same sequence has different genes (as shown in the figure below), is it because you simulated the sequence twice with ART?
![Uploading image.png…]()

**Comment by XLOXL** [comment]
I would like to ask you about the 150bp dataset of MAGs dataset, the subsequence with the same label in the same sequence has different genes (as shown in the figure below), is it because you simulated the sequence twice with ART?

![捕获](https://github.com/MicrobeLab/DeepMicrobes-data/assets/81890558/cc256c20-1097-430c-99c1-d3c1dc514767)

**Comment by MicrobeLab** [comment]
Hi, if I remember correctly, that's because I also got the reverse complementary sequence for each read. By the way, each sequence is just a simulated read, not a gene.

**Comment by XLOXL** [comment]
First of all, thank you for pointing out my mistakes. When I use the directive: art_illumina-ss HS25-i MAG-GUT2123. fa-p-l 150-f 2-m400-s 50-o out, I will only get a pair of reads NODE_45_length_142245_cov_5.87981-1244/1 and NODE_45_length_142245_cov_5.87981-1244/2, And why do you get two pairs in your data set namely two NODE_45_length_142245_cov_5.87981-1244/1 and two NODE_45_length_142245_cov_5.87981-1244/2(as shown in the picture above).

**Comment by MicrobeLab** [comment]
The two sequences with the same name are reverse complementary.
